### ISSUE TITLE
ImportError: No module named expat; use SimpleXMLTreeBuilder instead

### ISSUE THREAD

**Issue by BradCrumb** [bradfeehan/SublimePHPCoverage#14]
Hi there,

I got an error if I use the Plugin on Linux.

ImportError: No module named expat; use SimpleXMLTreeBuilder instead

I tried to symlink the original python2.6 libs to the sublime lib folder, but still no luck.

When I use the plugin on OSX it works fine.

Thanks

**Comment by bartnuman** [comment]
I have the same problem.
Using Xubuntu 13.04 32bit

**Comment by bradfeehan** [comment]
Hey, thanks for the bug report.

Up until now I've only tested it on Mac OS X. Honestly when I posted this up I didn't expect anybody else to use it but me!

I've just spent the last couple of weekends hacking away at a big refactor of the extension (#15), and I figured it'd be a good idea to do it properly and test on other platforms and machines. Unfortunately I don't have a Linux desktop to test on, so I'll obviously have to look into it a bit. I'll hopefully be able to get a VM set up and running to test shortly.

Until then (at least next weekend), if anybody else wants to tackle it, I think the refactor should make it a whole lot easier. If not, I'll definitely get around to it eventually.
